docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -1,48 +1,36 @@
 ---
+title: Extend the Kubernetes API with CustomResourceDefinitions
 assignees:
-- IanLewis
-title: Extending the Kubernetes API Using Custom Resource Definitions
-redirect_from:
-- "/docs/user-guide/customresourcedefinitions/"
-- "/docs/user-guide/customresourcedefinitions.html"
-- "/docs/concepts/ecosystem/customresourcedefinitions/"
-- "/docs/concepts/ecosystem/customresourcedefinitions.html"
+- deads2k
+- enisoc
 ---
 
-* TOC
-{:toc}
+{% capture overview %}
+This page shows how to install a [custom resource](/docs/concepts/extensions/custom-resources/)
+into the Kubernetes API by creating a CustomResourceDefinition.
+{% endcapture %}
 
-## What is a CustomResourceDefinition?
+{% capture prerequisites %}
+* Read about [custom resources](/docs/concepts/extensions/custom-resources/).
+* Make sure your Kubernetes cluster has a master version of 1.7.0 or higher.
+{% endcapture %}
 
-Kubernetes comes with many built-in API objects. However, there are often times
-when you might need to extend Kubernetes with your own API objects in order to do custom automation.
+{% capture steps %}
+## Create a CustomResourceDefinition
 
-`CustomResourceDefinition` objects are a way to extend the Kubernetes API with
-a new API object type. The new API object type will be given an API endpoint
-URL and support CRUD operations, and watch API. You can then create custom
-objects using this API endpoint. You can think of `CustomResourceDefinitions`
-as being much like the schema for a database table. Once you have created the
-table, you can then start storing rows in the table. Once created,
-`CustomResourceDefinitions` can act as the data model behind custom controllers
-or automation programs.
-
-A `CustomResourceDefinition` creates the REST API for a custom resource of your chosen name.
-
-## Creating a CustomResourceDefinition
-
-When you create a new `CustomResourceDefinition`, the Kubernetes API Server
-reacts by creating a new RESTful resource path (namespaced or cluster-scoped)
+When you create a new *CustomResourceDefinition* (CRD), the Kubernetes API Server
+reacts by creating a new RESTful resource path, either namespaced or cluster-scoped
```

Comment:
this isn't part of your change, but might it be clearer to say "either namespaced or cluster-scoped, as specified in the CRD's YAML config"? It wasn't immediately obvious to me what "request" meant in the next line.